smart-contracts/erc20/assembly/interfaces/erc20Core.ts

```diff
@@ -0,0 +1,14 @@
+import { u256 } from "as-bignum/assembly";
+import { Address } from "@massalabs/massa-as-sdk";
+
+export interface ERC20Core {
+    allowance(owner: Address, spender: Address): u256;
+    approve(spender: Address, value: u256): void;
+    balanceOf(address: Address): u256;
+    // decimals(): u8;
+    // name(): string;
+    // symbol(): string;
```

Comment:
why commented?

Reply:
Because if I dont do this, the IDE yells at me because those methods are missing from the ERC20 implementation.